packages/web-react/src/components/Footer/demo/FooterLinkFactory.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+type FooterLinkFactoryProps = {
+  items: number;
+  label: string;
+};
+
+const FooterLinkFactory = ({ items, label }: FooterLinkFactoryProps) => (
+  <>
+    {Array.from({ length: items }, (_, index) => (
```

Comment:
Are you sure this is working? I think only `map` can be used to render lists.

https://react.dev/learn/rendering-lists

Reply:
It seems to be working fine. I copied what we already have in `GridItemFactory.tsx` to maintain the same approach across the code.

Reply:
Ok.